### PR TITLE
Update database-initialization.rst

### PR DESCRIPTION
**Pull request by TriformDaniel**
Just a small start to make the installation easier.

### CODE REVIEW

File: en/installation-guide/database-initialization.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can use :ref:`pgAdmin` to access and manage the database server.
 Process
 -------
 
-In pgAdminIII
+In pgAdminIII or pgAdmin4
```
Comment (m-kuhn) [inline en/installation-guide/database-initialization.rst]: I think we could switch everything to pgAdmin4. We better have one clear recommendation (and those who want to stick to pgAdmin3 can do that anyway)

File: en/installation-guide/database-initialization.rst [hunk 2]
```diff
@@ -67,6 +67,10 @@ Restore the latest data model that also includes demo data:
   .. figure:: images/qgep_restore_messages.jpg
   
   * Check whether in Message window last line is Exit Code 0
+  If the Exit Code is 1 and the Detail reads sth like
```
Comment (m-kuhn) [inline en/installation-guide/database-initialization.rst]:
```suggestion
.. note::

  If the Exit Code is 1 and the Detail reads sth like
```

Comment (m-kuhn) [reply en/installation-guide/database-initialization.rst]: @TriformDaniel above I wrote a suggestion to change this to a "note". You can accept it if you like.

**Comment by m-kuhn** [comment]
@sjib there were open questions on this pull request. How should we proceed with these?

**Comment by sjib** [comment]
I tried to resolve them. But I missed this line:

 If the Exit Code is 1 and the Detail reads sth like

Can you add it?

File: en/installation-guide/database-initialization.rst (hunk 1 above)
Comment (sjib) [reply en/installation-guide/database-initialization.rst]: ok, should be done - just leaving the version away.

**Pull request merged**